index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="desktop-style.css" />
+    <link rel="stylesheet" href="mobile-style.css" />
+    <title>Capstone</title>
+  </head>
+  <body>
+    <header class="black-menu hide">
+      <ul class="black-menu hide">
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/facebook.png"
+              alt="facebook-logo"
+              height="20"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/twitter.png"
+              alt="twitter-logo"
+              height="20"
+          /></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </header>
+
+    
+    <div class="menu">
+      <a href="#" class="disapeer" id="hamburger-menu"
+        ><img
+          class="disapeer"
+          src="Images/menu-mobile.png"
+          alt="hamburger-menu"
+          width="60"
+      /></a>
+      <a class="link hide" href="index.html"
+        ><img
+          class="hide right"
+          src="Images/logo-villa.png"
+          alt="logo"
+          width="135"
+      /></a>
+      <ul class="menu-section">
+        <li>
+          <span class="material-icons close disapeer">close</span>
+        </li>
+        <li><a class="link" href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="cc"><a href="#">CC Campaign</a></li>
+      </ul>
+    </div>
+    <main>
+      <section class="headline" id="homePage">
+        <div class="text">
+          <h3 class="small-title">Welcome to the amazing</h3>
+          <h1 class="big-title">Villa Iluminada Atlixco, Puebla 2021</h1>
+          <p class="paragraph">
+            This tradition started in 2011 and ever since, it has been one of
+            the most awaited events in the locality. People from other states of
+            the country come each December to admire the beauty and the magic of
+            the lights.
+          </p>
+          <p class="last-info">2021.11.27(SAT) ~ 2022.01.06(THU)</p>
+          <p class="last-info">@villailuminada Atlixco,Pue.</p>
+        </div>
+      </section>
```

Comment:
- [ ] The typography (font family, font-weight, font size,...), headings texts, paragraphs, and spacing contents of your _hero_/ _banner_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Once again, we actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** matches that of the provided template design

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/133910014-adced48d-62db-437a-a04a-ce16f4e4e63a.png)

**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)

Reply:
- [ ] <b>REQUIRED:</b> As mentioned in the previous review, there are still few things to fix in your banner content to make it look more like the banner in mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| ![scrnli_9_19_2021_4-37-30 AM](https://user-images.githubusercontent.com/45217364/133913604-482a0678-67f7-4cf6-b66d-de3045b26dd4.png) | ![scrnli_9_19_2021_4-37-05 AM](https://user-images.githubusercontent.com/45217364/133913608-8ef83716-a42a-41fe-87e3-ee617d70e951.png) |
  - Just focus on text-color(you can simply use this exentension called [color picker](https://chrome.google.com/webstore/detail/colorpick-eyedropper/ohcpnigalekghcmgcdcenkpelffpdolg) to pick the exact color front the design), font-weight and font size

Reply:
> * [x]  The typography (font family, font-weight, font size,...), headings texts, paragraphs, and spacing contents of your _hero_/ _banner_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Once again, we actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** matches that of the provided template design
> 
> **Your hero design:**
> ![image](https://user-images.githubusercontent.com/12745474/133910014-adced48d-62db-437a-a04a-ce16f4e4e63a.png)
> 
> **The original hero design:**
> ![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)

- This would greatly help improve your project 💯  if addressed 👍